activiti-core/activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/process/ProcessRuntimeIT.java

```diff
@@ -268,6 +271,55 @@ public void createProcessInstanceAndValidateHappyPath() {
         assertThat(RuntimeTestConfiguration.discardImageConnectorExecuted).isEqualTo(false);
     }
 
+    @Test
+    public void createLinkEventProcessInstanceAndValidateHappyPath() {
+        //when
+        ProcessInstance linkProcess = processRuntime.start(ProcessPayloadBuilder.start()
+            .withProcessDefinitionKey(LINK_PROCESS)
+            .build());
+
+        //then
+        assertThat(RuntimeTestConfiguration.completedProcesses).contains(linkProcess.getId());
+        assertThat(RuntimeTestConfiguration.completedEvents).contains("throwEvent","intermediateCatchEvent");
+    }
+
+    @Test
+    public void createLinkEventProcessInstanceWithInclusiveGatewayAndValidateHappyPath() {
+        //when
+        ProcessInstance linkProcess = processRuntime.start(ProcessPayloadBuilder.start()
+            .withProcessDefinitionKey(LINK_PROCESS_WITH_GATEWAY)
+            .build());
+
+        TaskQuery query = taskService.createTaskQuery().orderByTaskName().asc();
+        var tasks = query.list();
+
+        assertThat(tasks).hasSize(2);
```

Comment:
This assertion can be removed as `containsExacly` will already fail if it has less or more elements. Same for similar assertions here.